docs/reference/commandline/tag.md

```diff
@@ -16,5 +16,11 @@ parent = "smn_cli"
 
       --help               Print usage
 
+Image name must match the regular expression `[a-z0-9]+`. 
```

Comment:
Hm, I think that's correct, yes.

(but @stevvooe or @aaronlehmann are the experts in conventions for these names :smile:)